.devcontainer/devcontainer.json

```diff
@@ -6,5 +6,13 @@
   "features": {
     "ghcr.io/devcontainers/features/docker-in-docker": {}
   },
+  "mounts": [
+    // To enable kernel modules for devcontainer
+    "source=/lib/modules,target=/lib/modules,type=bind,readonly",
+    "source=/usr/lib/modules,target=/usr/lib/modules,type=bind,readonly"
+  ],
+  "runArgs": [
+    "--sysctl", "net.ipv6.conf.all.disable_ipv6=0"
```

Comment:
without this, there will be system config conflict for IPv6. that is going to lead to the following error.

```bash
level=warning msg="Waiting for k8s node information" error="required IPv6 PodCIDR not available" subsys=daemon
```